src/components/ContextBar.js

```diff
@@ -43,6 +44,7 @@ export const ContextBar = ({
 }) => {
   const [isCloudEnvOpen, setCloudEnvOpen] = useState(false)
   const [selectedToolIcon, setSelectedToolIcon] = useState(undefined)
+  const [computeRegion] = useState(getRegionInfo(location, locationType).computeRegion)
```

Comment:
do we need to use `useState` for this if it is never updated via a setter, or can it be a regular `const` variable?